torch/csrc/jit/codegen/cuda/runtime/welford.cu

```diff
@@ -353,9 +353,9 @@ __device__ bool gridWelford(
     T& out_M2,
     T& out_avg,
     TN& out_N,
-    T inp_M2,
-    T inp_avg,
-    TN inp_N,
+    const T& inp_M2,
+    const T& inp_avg,
+    const TN& inp_N,
```

Comment:
@naoyam Thanks for helping fixing this. There seems to be another one at line40, in `blockWelford`.

Reply:
Changed it to a const reference. Thanks.